### PR TITLE
docs: Add configmap docs

### LINKED ISSUE #590: Document configmap
**Describe the solution you'd like**
The configmap schema should be documented, including the schema for the provided trivy scanner.

**Environment:**

- Eraser version: v1.0.0-beta.3
- Kubernetes version: (use `kubectl version`):

### PR DESCRIPTION
**Pull request by pmengelbert**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #590 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by ashnamehrotra** [comment]
@pmengelbert should we also change the flags in the documentation from the metrics, trivy, and FAQ section?

**Comment by pmengelbert** [comment]
@ashnamehrotra Yes, thank you!

File: docs/versioned_docs/version-v1.0.x/installation.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ title: Installation
 To install Eraser with the manifest file, run the following command:
 
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/Azure/eraser/v1.0.0/deploy/eraser.yaml
+kubectl apply -f https://raw.githubusercontent.com/Azure/eraser/master/deploy/eraser.yaml
```
Comment (sozercan) [inline docs/versioned_docs/version-v1.0.x/installation.md]: why are we changing to master? right now, this is pointing to beta3. we need to update with v1.1.0-beta.0

Comment (pmengelbert) [reply docs/versioned_docs/version-v1.0.x/installation.md]: Not sure how that happened, it's a mistake. I'll revert it

**Comment by pmengelbert** [comment]
Added v1.1.x docs in addition to updating v1.0.x. @ashnamehrotra the cli flags have been replaced in the other sections

File: docs/versioned_docs/version-v1.1.x/architecture.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (sozercan) [inline docs/versioned_docs/version-v1.1.x/architecture.md]: we don't need v1.1 docs yet, this should be done after cutting a release

Comment (pmengelbert) [reply docs/versioned_docs/version-v1.1.x/architecture.md]: Done, thanks

**Review by sozercan — APPROVED** [review]
Thank you! LGTM

**Pull request merged**